.editorconfig

```diff
@@ -15,6 +15,11 @@ indent_size = 2
 # C# files
 [*.cs]
 
+[*.go]

```

Comment:
In go, go fmt is universally used to format go programs. It doesn't enforce many style rules, but one rule it does enforce is that tabs with size 8 are used. If I don't add this to the editorconfig, go fmt complains. 

Note 1: we use gofumpt instead of go fmt. gofumpt enforces all the same style rules as go fmt, plus some nice extras
Note 2: if the tab size is 4, for some reason gofumpt (and go fmt) only complain about the tab size if there is also another style issue in the file.

Reply:
Weird, this problem seemed to go away for me, so I'll remove these changes